packages/leva/src/components/Leva/Filter.tsx

```diff
@@ -83,13 +90,35 @@ export function TitleWithFilter({
     return () => window.removeEventListener('keydown', handleShortcut)
   }, [])
 
+  const [copied, setCopied] = useState(false)
+  const handleCopyClick = async () => {
+    const data: any = store.getData()
```

Comment:
You shouldn't mutate the store data directly I'm afraid, I think this is a zustand thing more than Leva. If you look at the logs when copying and switching stories afterwards you'll notice errors.

Reply:
Agree. I actually didn't even notice that I did that. My goal with this PR was to just be a POC to "prove my concept" I mentioned on the #189.

In this case here, I'm not sure if that is the best way to do get a snap of the store. I'm not familiar with the code base so I might break something if I keep doing changes 😂